tests/Doctrine/Tests/DoctrineTestCase.php

```diff
@@ -2,9 +2,15 @@
 
 namespace Doctrine\Tests;
 
+// backward compatibility
```

Comment:
@GrahamCampbell This was an attempt to fix the travis build errors. It was not meant to be included in the pull request.